### ISSUE TITLE
🐞 - `DropdownContext` broken keyboard navigation

### ISSUE THREAD

**Issue by nsbarsukov** [Tinkoff/taiga-ui#2523]
### Which @taiga-ui/* package(s) are the source of the bug?

kit

### Is this issue blocking you?

Non-Blocking

### Description

1. Open https://taiga-ui.dev/directives/dropdown-context#contextMenu
2. Make right-click on any row
3. Press ArrowUp

**Expected behaviour:**
<img width="879" alt="Screenshot 2022-08-30 at 16 01 32" src="https://user-images.githubusercontent.com/35179038/187443376-ebe9d701-ae24-4ebf-b3e4-71bdd0673747.png">

**Notes:** it is regression bug after v3.0 release (it properly work in v2.x)

### Angular version

any

### Taiga UI version

3.0.0

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

**Comment by waterplea** [comment]
I think I purposefully omitted it because code was really complex and with right click we definitely use mouse and not that much rely on keyboard 🤔

**Comment by waterplea** [comment]
Rest of keyboard navigation works, I'll close it for the lack of necessity.
